src/components/Briefs/BioInsights.tsx

```diff
@@ -57,7 +59,10 @@ const BioInsights = ({
             >
               Submit a Proposal <FaRegShareSquare />
             </button>
-            <button className="primary-btn primary-btn-outlined max-width-1100px:w-full max-width-500px:w-auto">
+            <button
+              onClick={() => saveBrief && saveBrief?.()}
```

Comment:
saveBrief() doesn't work?

Reply:
Please add attribute `disabled`

Reply:
its not a required prop .... so this prevents crashing if its not passed